### PR TITLE
Implementation: [Android] Show a special notice to users in Indonesia 

### PR DESCRIPTION
**Pull request by anikiki**
Task/Issue URL: https://app.asana.com/0/1200581511062568/1208125695673129/f

### Description
Added a new section in the New Tab Page and a view in the Legacy Tab Page.

### Steps to test this PR
See https://app.asana.com/0/0/1208180779804579/f

### UI changes
See https://app.asana.com/0/1200581511062568/1208180779804556/f

### CODE REVIEW

**Comment by anikiki** [comment]
* **#4933** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/4933?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @anikiki and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: app/src/main/java/com/duckduckgo/app/browser/indonesiamessage/IndonesiaNewTabSectionDataStore.kt [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    )
+
+    override val showMessage: Flow<Boolean>
+        get() = context.indonesiaNewTabSectionDataStore.data
+            .map { prefs ->
+                prefs[SHOW_MESSAGE] ?: false
+            }
+            .distinctUntilChanged()
+
+    override suspend fun updateShowMessage(maxCount: Int) {
```
Comment (malmstein) [inline app/src/main/java/com/duckduckgo/app/browser/indonesiamessage/IndonesiaNewTabSectionDataStore.kt:68]: this should be tested

File: app/src/main/java/com/duckduckgo/app/browser/indonesiamessage/IndonesiaNewTabSectionView.kt [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+            show()
+            viewModel.onMessageShown()
+        }
+    }
+}
+
+@ContributesActivePlugin(
+    AppScope::class,
+    boundType = NewTabPageSectionPlugin::class,
+    priority = -1,
```
Comment (malmstein) [inline app/src/main/java/com/duckduckgo/app/browser/indonesiamessage/IndonesiaNewTabSectionView.kt:116]: this should be moved to `NewTabPageSectionPlugin.PRIORITY_INDONESIA_MESSAGE` with a value of 5.

**Review by malmstein — APPROVED** [review]
Works as intented! Just a couple of non blockers, nice work!

File: app/src/main/java/com/duckduckgo/app/browser/indonesiamessage/IndonesiaNewTabSectionDataStore.kt (hunk 1 above)
Comment (anikiki) [reply app/src/main/java/com/duckduckgo/app/browser/indonesiamessage/IndonesiaNewTabSectionDataStore.kt:63]: Missed that, thanks! Added tests :+1:

File: app/src/main/java/com/duckduckgo/app/browser/indonesiamessage/IndonesiaNewTabSectionView.kt (hunk 2 above)
Comment (anikiki) [reply app/src/main/java/com/duckduckgo/app/browser/indonesiamessage/IndonesiaNewTabSectionView.kt]: Updated :+1:

**Pull request merged**